pkg/reconciler/instances/eventing/preaction/removenatsoperatorstep.go

```diff
@@ -63,25 +66,36 @@ func (r *removeNatsOperatorStep) Execute(context *service.ActionContext, logger
 	//	return nil
```

Comment:
I'd say not required for your PR, but if you could remove all these commented out code in these two files, it would be nice! :-)

Reply:
Done